Script/AtomicEditor/ui/modal/build/BuildSettingsWindow.ts

```diff
@@ -41,7 +41,7 @@ export class BuildSettingsWindow extends ModalWindow {
         platformSource.addItem(new Atomic.UISelectItem("Windows", "WindowsBuildSettings", "LogoWindows"));
         platformSource.addItem(new Atomic.UISelectItem("Mac", "MacBuildSettings", "LogoMac"));
         platformSource.addItem(new Atomic.UISelectItem("Android", "AndroidBuildSettings", "LogoAndroid"));
-        platformSource.addItem(new Atomic.UISelectItem("iOS", "iOSBuildSettings", "LogoIOS"));
+        platformSource.addItem(new Atomic.UISelectItem("iOS", "iosBuildSettings", "LogoIOS"));
```

Comment:
I made a comment above, I think consistent case for iOS, so:

iosBuildSettings and licenseIOS read the best...